hathorlib/base_transaction.py

```diff
@@ -393,6 +396,24 @@ def is_nft_creation(self) -> bool:
         """Returns True if it's an NFT creation transaction"""
         return False
 
+    def is_standard(self, max_output_script_size: int = settings.MAX_OUTPUT_SCRIPT_SIZE,
+                    allow_non_standard_script: bool = False) -> bool:
+        """Return True is the transaction is standard
+        """
+        is_nft_creation = self.is_nft_creation
+        # First we check if any output script exceeds the maximum script size allowed
+        # Then we must check if all outputs are standard, and we allow a non
+        # standard output only in nft creation transactions
+        for output in self.outputs:
+            if len(output.script) > max_output_script_size:
```

Comment:
I think the name of the variable `max_output_script_size` is misleading since the max_output_script_size is 1024 in hathor-core.

Reply:
I agree with you, will change to `stdtx_max_output_script_size` as suggested by @jansegre